Users/Glob_Patterns.rst

```diff
@@ -40,7 +40,9 @@ The special characters used in shell-style wildcards are:
 .. note::
 
     If you're looking for a negation pattern to exclude paths, check out the
-    ``--ignore`` argument or ``ignore`` .coafile option!
+    ``--ignore`` argument or ``ignore`` .coafile option `here
+    <http://coala.readthedocs.io/en/latest/Users/Tutorial.html#ignoring
```

Comment:
@Adrianzatreanu Why is there a new line after `here` above ? My vim editor shows properly indented code here : 
![glob_pattern](https://cloud.githubusercontent.com/assets/14862976/19190464/45e8a1d2-8cbb-11e6-8f4c-1640d401a57d.png)
Is the P.R ready to be merged ?